cf-monitord/history.c

```diff
@@ -74,7 +74,7 @@ static void PutRecordForTime(CF_DB *db, time_t time, const Averages *values)
     WriteDB(db, timekey, values, sizeof(Averages));
 }
 
-static void Nova_SaveFilePosition(const char *handle, char *name, long fileptr)
+static void Nova_SaveFilePosition(const char *handle, const char *name, long fileptr)
```

Comment:
These are arguments not variables. :wink:

Reply:
? you want me to remove const in from of char *name?

Reply:
No, just change the commit message to say *arguments* instead of *variables*.

Reply:
ACK, fixed commit message, squashed a bit.